boards/esp32-ethernet-kit/include/periph_conf.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ * Copyright (C) 2020 Google LLC
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_esp32_esp-ethernet-kit
+ * @brief       Peripheral MCU configuration for Espressif ESP32-Ethernet-Kit
+ * @file
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @author      Erik Ekman <eekman@google.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name    ADC and DAC channel configuration
+ * @{
+ */
+
+/**
+ * @brief   Declaration of GPIOs that can be used as ADC channels
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the ```adc_init``` function, they can be used for other
```

Comment:
I know you got it from another body, but we can do it correctly now.
```suggestion
 * channels with the `adc_init` function, they can be used for other
```